.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+{
+  "containerEnv": {
+    "TZ": ""
+  },
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+  "build": {
+    "dockerfile": "../Dockerfile",
+    "context": ".."
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "bajdzis.vscode-database", // Supports connections to mysql or postgres, over SSL, socked
```

Comment:
A lot of these are either not relevant to all golang coding, or they are more opinionated that we want to be.  I would suggest trimming this down to just the immediately relevant extensions that anyone would definitely want (which I think is just `golang.go`?)